src/core/server/ui_settings/ui_settings_config.ts

```diff
@@ -29,15 +29,7 @@ const deprecations: ConfigDeprecationProvider = ({ unused, renameFromRoot }) =>
 const configSchema = schema.object({
   overrides: schema.object(
     {
-      defaultRoute: schema.maybe(
-        schema.string({
-          validate(value) {
-            if (!value.startsWith('/')) {
-              return 'must start with a slash';
-            }
-          },
-        })
-      ),
+      defaultRoute: schema.maybe(schema.string()),
```

Comment:
Shouldn't we remove it completely? We validate overrides on the `start` https://github.com/elastic/kibana/blob/7296932f2d1c8ae065d7c6f3fec99d298bc07bf4/src/core/server/ui_settings/ui_settings_service.ts#L76
That's not the same thing entirely, but I would avoid spreading logic to different places.